data_retriever/migration_event_queue.py

```
@@ -0,0 +1,204 @@
+        self._migration_id = ""
+
+    def _send_status(self, status):
+        """
+        Send a migration status in log to notify of the current status of the migration
+        Args:
+            status (str): The migration status
+        Raises:
+            EventQueueException: If status could not be sent
+        """
+        if not self._conn or not self._cursor:
+            raise EventQueueException(f"Postgres connection not established")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)